scripts/asm-installer/tests/script_version_check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -eu
+
+while read -r KEYWORD; do
+  INSTALL_ASM_LINE="$(grep "^${KEYWORD}=" "${INSTALL_ASM_SCRIPT}")"
```

Comment:
I'd set `INSTALL_ASM_SCRIPT=install_asm; readonly INSTALL_ASM_SCRIPT;` in this file instead of passing them in by env. They're extremely unlikely to change, and if they do then the tests are going to fail anyways, and it would be easier to find the cause of the error.

Reply:
Done.